ex1.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
+
+
+vector = np.array(range(1, 21))
+vector[8:15] *= -1
+print(vector)
```

Comment:
File should end with a newline - 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline